packages/ketcher-core/src/utilities/KetcherLogger.ts

```diff
@@ -0,0 +1,77 @@
+export enum LogLevel {
+  ERROR = 0,
+  WARN = 1,
+  LOG = 2,
+}
+
+export interface LogSettings {
+  enabled?: boolean;
+  showTrace?: boolean;
+  level?: LogLevel;
+}
+
+export class KetcherLogger {
+  static get settings(): LogSettings {
+    if (!window?.ketcher) {
+      throw new Error(
+        'Ketcher needs to be initialized before KetcherLogger is used',
+      );
+    }
+
+    return window.ketcher.logging;
+  }
+
+  static set settings(newSettings: LogSettings) {
+    for (const [settingName, settingValue] of Object.entries(newSettings)) {
+      this.settings[settingName] = settingValue;
+    }
+  }
+
+  static log(message: unknown): void {
+    if (!this.isMinimumLogLevel(LogLevel.LOG)) {
+      return;
+    }
+
+    const { showTrace } = this.settings;
+
+    if (showTrace) {
+      window.console.trace(message);
+    } else {
+      window.console.log(message);
+    }
+  }
+
+  static warn(warning: unknown): void {
+    if (!this.isMinimumLogLevel(LogLevel.WARN)) {
+      return;
+    }
+
+    window.console.warn(warning);
+  }
+
+  static error(error: unknown): void {
```

Comment:
Could you, please, clarify the difference between `error` and `showExceptionLocation`? What is the difference between error and exception?  
My proposal is to have only one method `error`, which can work like this:
`KetcherLogger.error('someFile::someFunction', error);` – please, note that error can be an object

Reply:
Done